test/integration/targets/bigip_device_auth_radius/tasks/main.yaml

```diff
@@ -0,0 +1,120 @@
+---
+
+- import_tasks: setup.yaml
+
+- name: Create an RADIUS device configuration
+  bigip_device_auth_radius:
+      servers:
+          - server: 1.1.1.1
+            port: 1812
+            secret: secret
+            timeout: 5
+      retries: 3
+      service_type: authenticate-only
+      accounting_bug: no
+      update_secret: on_create
+      use_for_auth: yes
+      state: present
+
+  register: result
+
+- name: Assert Create an RADIUS device configuration
+  assert:
+      that:
+          - result is success
+          - result is changed
+
+- name: Create an RADIUS device configuration - Idempotent check
+  bigip_device_auth_radius:
+      servers:
+          - server: 1.1.1.1
+            port: 1812
+            secret: secret
+            timeout: 5
+      retries: 3
+      service_type: authenticate-only
+      accounting_bug: no
+      update_secret: on_create
+      use_for_auth: yes
+      state: present
+  register: result
+
+- debug:
+      var: result
+      verbosity: 3
+
+- name: Assert Create an RADIUS device configuration - Idempotent check
+  assert:
+      that:
+          - result is success
+          - result is not changed
+
+- name: Update an RADIUS device configuration
```

Comment:
each check should have corresponding idempotency check

Reply:
add these after each test, this ensures that if you run with the same parameters the module won't change anything again

Reply:
This comment is addressed